diff --git a/templates/module/entity-content-page.php.twig b/templates/module/entity-content-page.php.twig
index d1092f6..d822290 100644
--- a/templates/module/entity-content-page.php.twig
+++ b/templates/module/entity-content-page.php.twig
@@ -9,6 +9,8 @@
 
 {% block use_class %}
 use Drupal\Core\Render\Element;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
 {% endblock %}
 
 {% block file_methods %}
